the-graph/the-graph-menu.js

```diff
@@ -85,9 +85,17 @@ function createMenuSlice(options) {
   var children = [
     factories.createMenuSliceArcPath(arcPathOptions)
   ];
+  var slice = this.props.menu[direction];
+  // If slice is defined and it's an object the flag will always be true.
+  var shouldShowSlice = !!slice;
```

Comment:
This is only checking that `slice` evaluates to `true`, could slice be defined to anything different from an object? In that case this could evaluate to `true`, invalidating the comment above.

Reply:
Gotcha!